pyqtgraph/opengl/GLViewWidget.py

```diff
@@ -381,14 +381,6 @@ def pan(self, dx, dy, dz, relative='global'):
         Prior to version 0.11, *relative* was expected to be either True (x-aligned) or
         False (global). These values are deprecated but still recognized.
```

Comment:
This documentation also needs to be removed

Reply:
Nice catch, removed!